gradio/pipelines_utils.py

```diff
@@ -26,21 +26,14 @@ def is_transformers_pipeline_type(pipeline, class_name: str):
     # version of the transformers library that the user has installed.
     if is_transformers_pipeline_type(pipeline, "AudioClassificationPipeline"):
         return {
-            "inputs": components.Audio(
-                sources=["microphone"],
-                type="filepath",
-                label="Input",
-                render=False,
-            ),
+            "inputs": components.Audio(type="filepath", label="Input", render=False),
```

Comment:
Audio sources shouldn't be limited to "microphone"?

Reply:
Agreed